### ISSUE TITLE
[minor-UX] Reflect Scope in Audit

### ISSUE THREAD

**Issue by yatil** [w3c/wcag-em-report-tool#318]
Totally minor, please don’t block publication.

When the scope is selected in Step 2, for example to WCAG 2.0 AA, this should be reflected in the Audit Sample step 4.

**Comment by nitedog** [comment]
Technically it does, but I think only if you go through the steps from 2 to 4, not the other way round. Something with the caching and the current framework we use. This may need to be fixed after we transition to a newer framework.

**Comment by rvantonisse** [comment]
Thanks for your input!

Yes easily implemented, but may be influenced by the decision made in #317
